Dockerfile

```diff
@@ -0,0 +1,52 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:20.04
+
+RUN apt-get update
+RUN apt-get install -y \
+    build-essential \
+    cgroup-lite \
+    cppreference-doc-en-html \
+    fp-compiler \
+    git \
+    haskell-platform \
+    libcap-dev \
+    libcups2-dev \
+    libffi-dev \
+    libpq-dev \
+    libyaml-dev \
+    mono-mcs \
+    openjdk-8-jdk-headless \
+    php7.4-cli \
+    postgresql-client \
+    python2 \
+    python3-pip \
+    python3.8 \
+    python3.8-dev \
+    rustc \
+    sudo \
+    wait-for-it \
+    zip
+
+# Create cmsuser user with sudo privileges
+RUN useradd -ms /bin/bash cmsuser && \
+    usermod -aG sudo cmsuser
+# Disable sudo password
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+# Set cmsuser as default user
+USER cmsuser
+
+COPY --chown=cmsuser:cmsuser . /home/cmsuser/cms
+
+WORKDIR /home/cmsuser/cms
+
+RUN sudo pip3 install -r requirements.txt
+RUN sudo pip3 install -r dev-requirements.txt
+RUN sudo python3 setup.py install
+
+RUN sudo python3 prerequisites.py --yes --cmsuser=cmsuser install
+
+RUN sudo cp config/cms-testdb.conf /usr/local/etc/cms-testdb.conf
```

Comment:
IMHO using `sed` is better than keeping a second copy of the file almost identical, that we'll have to remember to keep in sync.

Reply:
You're probably right, I changed it back to a sed. The main thing I wanted to achieve was to do "everything at build time" (i.e. in the Dockerfile) so that at runtime we just need to choose the config file to use. This is still possible with sed (running it in the Dockerfile) so we can do that.

Another possible approach here could be to remove the connection string from the config file and have it as an environment variable, which would make things a bit easier / cleaner, considering that IIRC in the future we want to replace the config files pretty much completely in favor of storing more and more settings in the DB.